prow/images/buildpack-golang-toolbox/Makefile

```diff
@@ -0,0 +1,15 @@
+IMG_NAME = buildpack-golang-toolbox
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(IMG_NAME)
+TAG = $(DOCKER_TAG)
+
+ci-pr: build-image push-image
+ci-release: build-image push-image-release
+
+build-image:
+	docker build -t $(IMG_NAME) .
+push-image:
+	docker tag $(IMG_NAME) $(IMG):$(TAG)
+	docker push $(IMG):$(TAG)
+push-image-release: push-image
+	docker tag $(IMG_NAME) $(IMG):go1.12-v1
```

Comment:
release tag `go:1.12-v1` is not clear. either tag something which will reveal this is more than just go, or you can just use push-image for releasing as well. It will then calculate the tag like [this](https://github.com/kyma-project/test-infra/blob/master/prow/scripts/publish-buildpack.sh#L24).